v1models/transform.go

```diff
@@ -33,6 +35,11 @@ func TransformProfileFromV2ToV1(profile models.DeviceProfile) (DeviceProfile, er
 	v1dp.DeviceCommands = toV1DeviceCommands(profile.DeviceCommands)
 	v1dp.CoreCommands = toV1CoreCommand(profile.DeviceResources, profile.DeviceCommands)
 
+	v1dp, err = convertStartingAddressToOneBased(v1dp)
```

Comment:
retrieve the first Device Resource, you only need to call this function if the attributes contain both "primaryTable" and "startingAddress"